modules/oe_content_call_proposals/src/CallForProposalsNodeWrapper.php

```diff
@@ -0,0 +1,159 @@
+<?php
```

Comment:
lets add a test for this, see https://github.com/openeuropa/oe_content/blob/master/modules/oe_content_call_tenders/tests/src/Kernel/CallForTendersNodeWrapperTest.php

Reply:
I added the test